app.js

```diff
@@ -1,10 +1,10 @@
-const { application } = require("express");
+const app  = express()
 const express = require("express");
 
 app.use(express.static("public"));
 
-app.get("/home", (request, response)=> {
+app.get("/", (request, response)=> {
     console.log(__dirname);
```

Comment:
The only page that is working is the home page, this is happening because you only have one route `/` you need to have one route for each page that you want to render in your website:

```
app.get("/", (request, response) => {
  response.sendFile(__dirname + "/view/home.html");
});

app.get("/about", (request, response) => {
  response.sendFile(__dirname + "/view/about.html");
});

app.get("/gallery", (request, response) => {
  response.sendFile(__dirname + "/view/photo.html");
});

app.get("/works", (request, response) => {
  response.sendFile(__dirname + "/view/works.html");
});
```

In the first line `app.get("/works", (request, response) => {` you specify the path that you want to appear in the `url` so in this case is `/works` this means that when you open your website in the browser and navigate to the `works` page your link will be `https://www.localhost:300/works`

In the second line `response.sendFile(__dirname + "/view/works.html");` you specify what to you want to show to the user when he navigates to the `/works` page so in this case you want to render the `html` file that is inside the folder `/view/works.html`

The path for the `url` and the path for the `html` file are two completely different things and they can be named differently.

Reply:
Hello João, i did all the changes but now works, photos and about gives an error. 
Can we check this tomorrow if possible?
thank you so much :)

Reply:
Are you starting the server with `nodemon app.js` on the terminal or are you using the live server in vscode?